Backend/lib/scheduler/src/Lib/Scheduler/JobStorageType/Redis/Queries.hs

```diff
@@ -0,0 +1,109 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
```

Comment:
can you remove this?

Reply:
> In the ambiguity check for ‘createJob’
>       To defer the ambiguity check to use sites, enable AllowAmbiguousTypes
>       In the type signature:
>         createJob :: forall t (e :: t) m r.
>                      (JobFlow t e,
>                       JobMonad m,
>                       HedisFlow m r,
>                       HasField "setName" r Text) =>
>                      Int -> JobContent e -> m ()



Removed `{-# OPTIONS_GHC -Wno-deprecations #-}`
 added `{-# LANGUAGE AllowAmbiguousTypes #-}` because of error above